vector/src/main/java/im/vector/app/core/platform/VectorBaseActivity.kt

```diff
@@ -345,6 +345,14 @@ abstract class VectorBaseActivity : AppCompatActivity(), HasScreenInjector {
                 }
     }
 
+    protected var postResumeScheduledAction: Runnable? = null
```

Comment:
Could u create a list instead? so we can handle multiple things

Reply:
No usage for the moment (so no need to do complicated things), but yes, why not...